components/Slider/src/MDCSlider.h

```diff
@@ -162,18 +162,33 @@ IB_DESIGNABLE
 - (nullable UIColor *)backgroundTrackTickColorForState:(UIControlState)state;
 
 /**
- The color of the Ink ripple.
+ Sets the thumb radius to use for the specified state.
 
- Defaults to transparent blue.
+ In general, if a property is not specified for a state, the default is to use the @c
+ UIControlStateNormal value. If the @c UIControlStateNormal value is not set, then the property
+ defaults to a default value. Therefore, at a minimum, you should set the value for the normal
+ state.
+
+ @param thumbRadius The radius of the thumb.
+ @param state The state of the slider.
  */
-@property(nonatomic, strong, nullable) UIColor *inkColor;
+- (void)setThumbRadius:(CGFloat)thumbRadius forState:(UIControlState)state;
```

Comment:
Can we please document what happens with negative or zero values?

Reply:
Have documented that if a client uses a negative value then the absolute value will be used.